src/types/common.d.ts

```diff
@@ -0,0 +1,22 @@
+export interface MusicAccidental {
```

Comment:
Do we need Music prefix here? I would suggest to change MusicAccidental to Accidental and so on for the others

Reply:
I agree that this is not needed.